tests/test_kubernetes_tools.py

```diff
@@ -4768,7 +4946,7 @@ def test_warning_big_bounce_default_config():
             job_config.format_kubernetes_app().spec.template.metadata.labels[
                 "paasta.yelp.com/config_sha"
             ]
-            == "config84789e0b"
+            == "configcfb8df3a"
```

Comment:
i...am not really sure why changing the models to the latest clientlib versions will cause a big bounce since nothing is actually changing otherwise :(